src/cirrus/docker.py

```diff
@@ -148,9 +176,78 @@ def _docker_build(path, tag, base_tag):
     LOGGER.info(stdout)
     image = find_image_id(base_tag)
     LOGGER.info("Image ID: {}".format(image))
+
+    if not is_docker_version_installed('1.17.0'):
```

Comment:
suggest making this version a BIG_OBVIOUS_CONSTANT at the top of the module with a brief desc of what it is controlling